### PR TITLE
deleteSignature fixture added

### PR DESCRIPTION
**Pull request by teya-teya**
Add deleteSignature in the fixture
Add settingEditSignature.js
Make additions in signPage.js and testData.js
Add test 'create signature'
Add test 'create and delete signature'

### CODE REVIEW

File: fixtures/base.js [hunk 1]
```diff
@@ -36,4 +36,20 @@ export const test = base.extend({
...
+
+            const editSignature = await signPage.clickEditSignatureAndGoEditSignaturePage();
+            await editSignature.clickBurgerMenuSignature();
+            await editSignature.clickDeleteDropItem();
+            await editSignature.clickButtonDelete();
+            await editSignature.clickSignSidebarLinkAndGoSignPage();
+        },
+        { scope: "test" },
+    ],
 });
```
Comment (SvetlanaKanel) [inline fixtures/base.js:55]: Please, add a test to the navigation.spec to check how your fixture works.

Comment (teya-teya) [reply fixtures/base.js:55]: Thanks, I've fixed everything!

**Comment by teya-teya** [comment]
> Could you please add test with implementation this fixture in .spec.js to see how it works
Added a creation and deletion test together.

File: tests/navigation.spec.js [hunk 2]
```diff
@@ -13,4 +13,45 @@ test('Navigate to login page', async ({ page }) => {
 test('Check login fixture', async ({ page, loginBusinessUser}) => { 
 
   await expect(page).toHaveURL(BASE_URL + URL_END_POINTS.signEndPoint);
-});
\ No newline at end of file
+});
+
+test('create signature', async ({page, loginBusinessUser, deleteSignature}) => {
+  await page.locator('.dropDownUser__wrapper').click()
```
Comment (SvetlanaKanel) [inline tests/navigation.spec.js:19]: Why don't you use POM? Please, use locators and methods from pages.

Comment (teya-teya) [reply tests/navigation.spec.js]: Okay, I did it.

**Review by SvetlanaKanel — APPROVED** [review]

**Pull request merged**